packages/trader/src/Stores/Modules/Trading/Helpers/__tests__/contract-type.spec.ts

```diff
@@ -0,0 +1,570 @@
+                                                },
+                                            ],
+                                            name: 'AUD/JPY',
+                                            symbol: 'frxAUDJPY',
+                                            times: {
+                                                close: ['23:59:59'],
+                                                open: ['00:00:00'],
+                                                settlement: '23:59:59',
+                                            },
+                                            trading_days: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri'],
+                                        },
+                                    ],
+                                },
+                                {
+                                    name: 'Minor Pairs',
+                                    symbols: [
+                                        {
+                                            events: [
+                                                {
+                                                    dates: 'Fridays',
+                                                    descrip: 'Closes early (at 20:55)',
+                                                },
+                                                {
+                                                    dates: '2023-12-25',
+                                                    descrip: 'Christmas Day',
+                                                },
+                                                {
+                                                    dates: '2024-01-01',
+                                                    descrip: "New Year's Day",
+                                                },
+                                            ],
+                                            name: 'AUD/AED',
+                                            symbol: 'frxAUDAED',
+                                            times: {
+                                                close: ['23:59:59'],
+                                                open: ['00:00:00'],
+                                                settlement: '23:59:59',
+                                            },
+                                            trading_days: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri'],
+                                        },
+                                    ],
+                                },
+                            ],
+                        },
+                    ],
+                },
+                echo_req: {
+                    trading_times: '2023-12-01',
+                },
+            })
+        ),
+    },
+}));
+
+jest.mock('_common/base/server_time', () => ({
+    get: jest.fn(() => ({
+        isBefore: jest.fn(),
+        clone: jest.fn(),
+        get: jest.fn(),
+    })),
+}));
+
+const serverTimeGetSpy = jest.spyOn(ServerTime, 'get');
```

Comment:
```suggestion
const server_time_get_spy = jest.spyOn(ServerTime, 'get');
```
it looks like an object is returned :)

Reply:
done